examples/quantization_aware_training/tensorflow/mobilenet_v2/main.py

```diff
@@ -0,0 +1,200 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import subprocess
+from pathlib import Path
+from typing import List
+
+import openvino as ov
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from rich.progress import track
+
+import nncf
+
+ROOT = Path(__file__).parent.resolve()
+WEIGHTS_URL = "https://huggingface.co/alexsu52/mobilenet_v2_imagenette/resolve/main/tf_model.h5"
+DATASET_CLASSES = 10
+
+
+def validate(model: ov.Model, val_loader: tf.data.Dataset) -> tf.Tensor:
+    compiled_model = ov.compile_model(model, device_name="CPU")
+    output = compiled_model.outputs[0]
+
+    metric = tf.keras.metrics.CategoricalAccuracy(name="acc@1")
+    for images, labels in track(val_loader):
+        pred = compiled_model(images.numpy())[output]
+        metric.update_state(labels, pred)
+
+    return metric.result()
+
+
+def run_benchmark(model_path: Path, shape: List[int]) -> float:
+    command = [
+        "benchmark_app",
+        "-m", model_path.as_posix(),
+        "-d", "CPU",
+        "-api", "async",
+        "-t", "15",
+        "-shape", str(shape),
+    ]  # fmt: skip
+    cmd_output = subprocess.check_output(command, text=True)  # nosec
+    print(*cmd_output.splitlines()[-8:], sep="\n")
+    match = re.search(r"Throughput\: (.+?) FPS", cmd_output)
+    return float(match.group(1))
+
+
+def get_model_size(ir_path: Path, m_type: str = "Mb") -> float:
+    xml_size = ir_path.stat().st_size
+    bin_size = ir_path.with_suffix(".bin").stat().st_size
+    for t in ["bytes", "Kb", "Mb"]:
+        if m_type == t:
+            break
+        xml_size /= 1024
+        bin_size /= 1024
+    model_size = xml_size + bin_size
+    print(f"Model graph (xml):   {xml_size:.3f} Mb")
+    print(f"Model weights (bin): {bin_size:.3f} Mb")
+    print(f"Model size:          {model_size:.3f} Mb")
+    return model_size
+
+
+###############################################################################
+# Create a Tensorflow model and dataset
+
+
+def center_crop(image: tf.Tensor, image_size: int, crop_padding: int) -> tf.Tensor:
+    shape = tf.shape(image)
+    image_height = shape[0]
+    image_width = shape[1]
+
+    padded_center_crop_size = tf.cast(
+        ((image_size / (image_size + crop_padding)) * tf.cast(tf.minimum(image_height, image_width), tf.float32)),
+        tf.int32,
+    )
+
+    offset_height = ((image_height - padded_center_crop_size) + 1) // 2
+    offset_width = ((image_width - padded_center_crop_size) + 1) // 2
+
+    image = tf.image.crop_to_bounding_box(
+        image,
+        offset_height=offset_height,
+        offset_width=offset_width,
+        target_height=padded_center_crop_size,
+        target_width=padded_center_crop_size,
+    )
+
+    image = tf.compat.v1.image.resize(
```

Comment:
Can we use https://www.tensorflow.org/api_docs/python/tf/image/resize instead of `tf.compat.v1.image.resize`?  Looks like unique param `align_corners=False` does not do anything with the image and it safe to migrate to the newest API

Reply:
Done